File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
var brandService=app.service("brandService",function ($http) {
    //1.查询所有
    this.findAll = ()=>{
        return $http.get("../brand/list.do");
    }
    //2.分页查询
    this.findByPage = (page,pagesize)=>{
        return $http.get("../brand/findByPage.do?page="+page+"&pagesize="+pagesize);
    }
    //3.分页带条件查询
    this.search = (page,pagesize,entity)=>{
        return $http.post("../brand/search.do?page="+page+"&pagesize="+pagesize, entity);
    }
    //4.保存商品
    this.save = (url,entity)=>{
        return $http.post(url,entity);
    }
    //5.根据id删除品牌
    this.delete = (brandIds)=>{
        return $http.get("../brand/delete.do?ids="+brandIds);
    }
})

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;

/**
 * @Author: Feng.Wang
 * @Company: Zelin.ShenZhen
 * @Description:
 * @Date: Create in 2019/4/22 11:39
 */
public interface BrandService {
    public List<TbBrand> findAll();

    PageResult findByPage(int page, int pagesize);

    PageResult search(int page, int pagesize, TbBrand brand);

    void add(TbBrand brand);

    void update(TbBrand brand);

    void delete(String[] ids);
}
